### ISSUE TITLE
Return outputs directly

### ISSUE THREAD

**Issue by codeplea** [cirla/tulipy#7]
Would it make more sense to return outputs directly for indicators returning a single output? Indicators returning multiple outputs could use a tuple.

This way, you can easily pass indicator outputs directly to other indicators. For example, a moving average crossover strategy is simply:

```Python
buy = ti.crossover(ti.sma(close,5.0), ti.sma(close,24.0))
```

instead of:

```Python
buy = ti.crossover(ti.sma(close,5.0)['sma'], ti.sma(close,24.0)['sma'])['sma']
```

There are only 8 indicators that return multiple outputs, so I think that this would be optimizing for the major use case.

Using tuples for multiple outputs seems easier too:

```Python
lower, middle, upper = ti.bbands(close, 14.0, 1.0)
```

I've implemented the change in [this branch](https://github.com/cirla/tulipy/tree/return).

**Comment by cirla** [comment]
Yes, I totally agree. Could you create a PR from your branch? My only two comments:

- I've started generating README.md from exporting README.ipynb as Markdown (then adding the Travis CI badge at the top) so I'd like to keep them in sync. For minor changes like this you can probably just edit the raw ipynb directly instead of having to install jupyter.
- This is complete personal preference and bikeshedding so feel free to ignore, but when an `if` returns I avoid the `else:` because some static code analyzers complain if there's no return at the function scope.

**Comment by codeplea** [comment]
I believe I've addressed your concerns in this PR #9

**Comment by cirla** [comment]
Awesome, thanks!
